python/src/aiconfig/editor/client/src/components/prompt/PromptsContainer.tsx

```diff
@@ -58,25 +58,23 @@ export default memo(function PromptsContainer(props: Props) {
         />
       )}
       {props.prompts.map((prompt: ClientPrompt, i: number) => {
+        const promptId = prompt._ui.id;
         const isAnotherPromptRunning =
           props.runningPromptId !== undefined &&
-          props.runningPromptId !== prompt._ui.id;
+          props.runningPromptId !== promptId;
+
         return (
           <Stack key={prompt._ui.id}>
             <Flex mt="md">
-              {!readOnly && (
-                <PromptMenuButton
-                  promptId={prompt._ui.id}
-                  onDeletePrompt={() => props.onDeletePrompt(prompt._ui.id)}
-                />
-              )}
```

Comment:
omg this had me confused so hard wondering wtf was going on, but this is the `PromptsContainer`, not `PromptContainer` :S

Reply:
I like this change, I feel it makes sense to put with the individual prompt component anyways